test/token/ERC721/ERC721.test.js

```diff
@@ -21,11 +21,11 @@ contract('ERC721Token', function (accounts) {
   const anyone = accounts[9];
```

Comment:
nit: account destructuring

Reply:
The whole ERC721 test suite needs work (#1148), account destructuring is the least of our concerns there :(